connectors/sources/servicenow.py

```diff
@@ -327,6 +434,111 @@ async def ping(self):
             logger.exception("Error while connecting to the ServiceNow.")
             raise
 
+    def _format_doc(self, data):
+        """Format document for handling empty values & type casting.
+
+        Args:
+            data (dict): Fetched record from ServiceNow.
+
+        Returns:
+            dict: Formatted document.
+        """
+
+        data = {key: value for key, value in data.items() if value}
+        data.update(
+            {
+                "_id": data["sys_id"],
+                "_timestamp": iso_utc(parser.parse(data["sys_updated_on"])),
+            }
+        )
+        return self.serialize(doc=data)
```

Comment:
nit: I think we should keep the `serialize` call separate from `format` as these are different concerns IMO.